docs/theming/developers.md

```diff
@@ -200,46 +207,47 @@ themable graphics
+We developed our theming system with web standards in mind, aiming to use
+CSS over JavaScript as much as possible. At the current state of browser technology,
+we found that some JavaScript is required for the system to work correctly.
+But shortly, we expect to be able to remove all or nearly all of that 
+JavaScript and have the theming system work entirely (or almost entirely) 
 through plain CSS. The following explains how the system currently operates, how
 element authors can hook into it, and what the future holds for theming.
 
 ### Current implementation - Context Protocol
 
-<abbr title="Red Hat Design System">RHDS</abbr>' Theming system is primarily 
-about styles. Tt currently relies on JavaScript to work, by way of the [context 
+<abbr title="Red Hat Design System">RHDS</abbr>’ theming system is primarily 
+about styles. It currently relies on JavaScript to work, via the [context 
 protocol][contextprotocol] developed by the web components community to support 
 passing data between components.
 
 Our system utilizes this protocol with the setting of the `color-palette` 
 attribute on a provider element which makes its context data (in our case, 
-`light` or `dark`) to it's children.  By doing so we can ensure accessible 
+`light` or `dark`) to its children. By doing so we ensure accessible 
 colors are applied given any possible change in context value higher up in the 
 <abbr title="Document Object Model">DOM</abbr> tree. 
 
-This is important not only for helping us maintain our own design guidelines, 
-but also for accessibility compliance and enabling great experiences for all our 
+This is important not only for helping us maintain our design guidelines but
+also for accessibility compliance and enabling great experiences for all our 
 users.
 
 The context protocol is enabled by two [reactive controllers][controllers]: the
 [provider controller][providersrc], and the [consumer controller][consumersrc].
-These controllers work in tandem to alternately provide the context data, or
-consume it as a child element. Custom elements can implement the consumer, the
-provider, or both, depending on the needs of that particular element.
+The provider and consumer controllers work together. The provider supplies the
+context data, while the consumer receives and uses it within child elements.
+Custom elements can implement the consumer, the provider, or both, depending
+on the needs of that particular element.
 
 #### Providers
 
 Custom Elements that implement the provider controller are elements that provide
-their own context, overriding that of their parent.  Elements such as
+their own context, overriding that of their parent. Elements such as
 `<rh-surface>`, `<rh-card>`, and `<rh-accordion>` are examples of such context
 providers. If a provider contains a set `color-palette` attribute, it will
 override any parent context, and pass its context on to its children.
```

Comment:
Delete the comma after "context."

Reply:
https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/8ae31707ea11f29ae967709bb95afebae9820f44